library/include/hiprand/hiprand.h

```diff
@@ -62,7 +62,7 @@
 #define HIPRAND_VERSION
 #endif
 
-#if defined(__HIP_PLATFORM_HCC__)
+#if defined(__HIP_PLATFORM_HCC__) || defined(__HIP_PLATFORM_AMD__)
```

Comment:
Approving but why not remove the HCC section?

Reply:
> Approving but why not remove the HCC section?

I thought it would be best to provide support for both during the deprecation process.  Is it currently safe to have only __HIP_PLATFORM_AMD__?